db/seeds.rb

```diff
@@ -32,7 +32,7 @@
 end.save!
 
 Location.find_or_initialize_by(
-  city_name: 'SanDiego'
+  city_name: 'San Diego'
```

Comment:
This probably should be its own commit fixing the data.

Reply:
A new commit within this pull request or a commit for a brand new PR?